### PR TITLE
Style Guide Updates

### PR DESCRIPTION
**Pull request by biyeun**
@NoahCarnahan

Just some updates to the style guide. Added stuff on crsipy forms, Bootstrap 3 migrations and views

### CODE REVIEW

File: corehq/apps/style/crispy.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ class Field(OldField):
     """Overrides the logic behind choosing the offset class for the field to
     actually be responsive (col-lg-offset-*, col-md-offset-*, etc). Also includes
     support for static controls.
+    todo since we forked crispy forms, this class is no longer necessary. http://manage.dimagi.com/default.asp?186372
```
Comment (dimagimon) [inline corehq/apps/style/crispy.py]: E501 line too long (117 > 115 characters)

File: corehq/apps/style/crispy.py [hunk 2]
```diff
@@ -67,3 +68,30 @@ def render(self, form, form_style, context, template_pack=None):
...
+        if not hasattr(self, 'attrs'):
+            self.attrs = {}
+
+        if 'css_class' in kwargs:
+            if 'class' in self.attrs:
+                self.attrs['class'] += " %s" % kwargs.pop('css_class')
+            else:
+                self.attrs['class'] = kwargs.pop('css_class')
+
+    def render(self ,form, form_style, context, template_pack=None):
```
Comment (dimagimon) [inline corehq/apps/style/crispy.py]: E203 whitespace before ','

Comment (dimagimon) [inline corehq/apps/style/crispy.py]: E231 missing whitespace after ','

File: corehq/apps/styleguide/example_forms.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+from django import forms
+from django.template.loader import render_to_string
```
Comment (dimagimon) [inline corehq/apps/styleguide/example_forms.py]: F401 'render_to_string' imported but unused

**Comment by NoahCarnahan** [comment]
@biyeun Looks good. I found two small typos, but content looks great!

**Comment by biyeun** [comment]
@NoahCarnahan updated

**Pull request merged**